aip/general/0122.md

```diff
@@ -99,15 +99,32 @@ An API **should** use the less-redundant form:
 users/vhugo1802/events/birthday-dinner-226
 ```
 
-In this situation, the _message_ is still called `UserEvent`; only the resource
-name is shortened.
+In this situation, the _message_ and _resource type_ are still called
+`UserEvent`; only the collection and resource identifiers in the pattern(s) are 
+shortened. Since the _resource type_ is not shortened, the `singular` and
+`plural` are similarly _not shortened_.
+
+```
+message UserEvent {
+  option (google.api.resource) = {
+    type: "example.googleapis.com/UserEvent"
+    // Only the collection & resource identfiers in the `pattern` are shortened.    
+    pattern: "projects/{project}/users/{user}/events/{event}"
+    singular: "userEvent"
+    plural: "userEvents"
+  };
+
+  string name = 1;
+}
+```
 
 **Note:** APIs wishing to do this **must** follow this format consistently
-throughout the API, or else not at all.
+throughout all of its `pattern`s defined and anywhere else the resource is
+referenced in the API, or else not at all.
 
 ### Resource ID segments
 
-A resource ID segment identifies the resource within its parent collection. In
+A resource ID segment identifiers the resource within its parent collection. In
```

Comment:
This no longer makes sense as a sentence... either revert, or rework. (My guess is that your eye spotted it and thought it was a typo, when it wasn't.)

Reply:
Yep idk what I was thinking...